app/components/media/media_card_component.rb

```diff
@@ -0,0 +1,12 @@
+class Media::MediaCardComponent < ApplicationComponent
```

Comment:
This namespacing comes off as a bit redundant to me. I think it also slightly misrepresents the intent of the component too, since it's not _just_ media (ie an image, a video etc). Considering the overall heirarchy, either

```rb
module Media
  class CardComponent
  end
end
```
or more simply
```rb
class MediaCardComponent; end
```
Feel like better choices to me. The second would probably be my preference but depends on a few other factors.